tests/ut_common_copytotarget.py

```diff
@@ -0,0 +1,27 @@
+"""
+The unit test check if copy_to_target(file_expr, target) works fine.
+
+"""
+import sys
+import os
+import shutil
+import tempfile
+import build_component
+
+def main():
+    # create a new temporary file and a new temporary directory.
+    temp = tempfile.NamedTemporaryFile()
+    tempdir = tempfile.mkdtemp()
+
+    # copy a temporary file to a temporary directory.
+    build_component.copy_to_target(temp.name, tempdir)
```

Comment:
How will we handle exceptions that `copy_to_target` raises?

(I'm thinking of things like specifying a target dir for which the user has insufficient write permissions.)